Library/Homebrew/cmd/audit.rb

```diff
@@ -195,12 +195,12 @@ def audit_formula_name
     full_name = formula.full_name
 
     if @@aliases.include? name
-      problem "Formula name is conflicted with existed aliases."
+      problem "Formula name conflicts with existing aliases."
       return
     end
 
     if !formula.core_formula? && Formula.core_names.include?(name)
-      problem "Formula name is conflicted with existed core formula."
+      problem "Formula name conflicts with existing core formulae."
```

Comment:
This is a really small nit, but this should probably remain `core formula` rather than `core formulae.` - The latter is the plural, but the audit will only address one formula at a time.